lisa/tools/lsblk.py

```diff
@@ -3,13 +3,24 @@
 
 import re
 from dataclasses import dataclass, field
-from typing import Dict, List, Optional
+from typing import Dict, List, Optional, Type
 
 from lisa.executable import Tool
-from lisa.operating_system import FreeBSD, Posix
+from lisa.operating_system import BSD, Posix
 from lisa.util import LisaException, find_patterns_groups_in_lines
 
 
+def get_size_in_bytes(size: int, size_unit: str) -> int:
```

Comment:
Because it's not used out of this file, still make it preview starts with underscore.

Reply:
fixed